### PR TITLE
feat: Home

### PR DESCRIPTION
**Pull request by aniketsingh13**
Screenshot 2022-02-16 013438
Screenshot 2022-02-16 013551

### CODE REVIEW

**Review by jeetbhalani15 — APPROVED** [review]
Looks great

File: style.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+.nav-brand {
+    align-self: center;
+}
+
+.nav-brand-a {
+    color: var(--offwhite);
+}
+
+.nav-input {
+    background-color: 1px solid black;
```
Comment (Naveenchandar) [inline style.css]: Please use hex code for consistency.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Aniket singh" />
+    <meta name="description" content="PlayX-shop e-commerce app for gaming devices." />
+    <link rel="stylesheet" href="https://playx-ui.netlify.app/styles/main.css" />
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/fa50d5d333.js" crossorigin="anonymous"></script>
```
Comment (Naveenchandar) [inline index.html]: Font awesome can be used by link tags. Script is not needed. Please once verify it.

File: style.css (hunk 1 above)
Comment (aniketsingh13) [reply style.css]: make the changes

File: index.html (hunk 2 above)
Comment (aniketsingh13) [reply index.html]: will change it

**Pull request merged**